src/broker/cas_execute.c

```diff
@@ -4914,7 +4914,7 @@ cur_tuple (T_QUERY_RESULT * q_result, int max_col_size, char sensitive_flag, DB_
   char *null_type_column = q_result->null_type_column;
   int err_code;
 
-  ncols = db_query_column_count (result);
+  ncols = q_result->num_column;
```

Comment:
it would be better add a comment to explain the value.

Reply:
This code was occurred a problem when executing the call and evaluation statement. So I will revert it.